### PR TITLE
Me: remove noticons

### PR DESCRIPTION
**Pull request by drw158**
This removes noticons in Me

affected areas:

- Security > Two-Step Authentication > Backup Codes
- Security > Two-Step Authentication > App Passwords
- My Profile > Profile Links
- Next Steps
- Notification Settings

I removed the usage of the ActionRemove component, so I think that component can be removed.

I'm pretty sure I formatted things wrong in the react files. Some help would be appreciated.

### Before

screen shot 2017-03-08 at 4 28 26 pm
screen shot 2017-03-08 at 5 03 12 pm
screen shot 2017-03-08 at 5 03 53 pm
screen shot 2017-03-08 at 5 04 48 pm
screen shot 2017-03-08 at 5 05 10 pm

### After

screen shot 2017-03-08 at 5 08 32 pm
screen shot 2017-03-08 at 5 08 22 pm
screen shot 2017-03-08 at 5 08 13 pm
screen shot 2017-03-08 at 5 08 07 pm
screen shot 2017-03-08 at 4 28 21 pm

### CODE REVIEW

**Comment by rickybanister** [comment]
I love you!

**Comment by folletto** [comment]
Checked in Chrome:

1. Security > Two-Step Authentication > Backup Codes — ✅ 
2. Security > Two-Step Authentication > App Passwords — ✅ 
3. My Profile > Profile Links — ✅ 
4. Next Steps — ✅ *
5. Notification Settings — ✅ 

All looks good! 👍

File: client/me/application-password-item/index.jsx [hunk 1]
```diff
@@ -58,10 +59,10 @@ const ApplicationPasswordsItem = React.createClass( {
 						}
 					</p>
 				</div>
-				<ActionRemove
-					className="application-password-item__revoke"
+				<Button borderless icon className="application-password-item__revoke"
```
Comment (mtias) [inline client/me/application-password-item/index.jsx]: `icon` is not a valid prop, you can omit.

Comment (drw158) [inline client/me/application-password-item/index.jsx]: Thanks for the review, fixed.

**Comment by rickybanister** [comment]
problem with rebasing?

**Comment by drw158** [comment]
I think I just didn't do it right.

**Comment by mtias** [comment]
@drw158 cleaned up history.

File: client/me/application-password-item/index.jsx [hunk 2]
```diff
@@ -58,10 +59,10 @@ const ApplicationPasswordsItem = React.createClass( {
 						}
 					</p>
 				</div>
-				<ActionRemove
-					className="application-password-item__revoke"
+				<Button borderless className="application-password-item__revoke"
 					onClick={ this.recordClickEvent( 'Remove Application Password Button', this.removeApplicationPassword ) }
-				/>
+				>
+				<Gridicon icon="cross" /></Button>
```
Comment (gwwar) [inline client/me/application-password-item/index.jsx]: Maybe fix the indentation here, for example:
```jsx
<Button>
    <Gridicon />
</Button>
```

File: client/me/application-password-item/index.jsx [hunk 3]
```diff
@@ -9,9 +9,10 @@ var React = require( 'react' ),
 /**
  * Internal dependencies
  */
-var ActionRemove = require( 'me/action-remove' ),
-	eventRecorder = require( 'me/event-recorder' ),
-	errorNotice = require( 'state/notices/actions' ).errorNotice;
+var eventRecorder = require( 'me/event-recorder' ),
+	errorNotice = require( 'state/notices/actions' ).errorNotice,
+	Gridicon = require( 'gridicons' ),
```
Comment (gwwar) [inline client/me/application-password-item/index.jsx]: `gridicons` is technically an external dep now. Do you mind moving this up to  ` External dependencies`. Same thing for the other files that import gridicons.

**Comment by drw158** [comment]
Thanks @gwwar, should be all fixed up.

**Comment by gwwar** [comment]
Thanks @drw158 feel free to :shipit:

**Pull request merged**